### ISSUE TITLE
Calibration Application - Gesture recording doesn't fill the green bar.

### ISSUE THREAD

**Issue by lhoward465** [ariawearable/Android-SDK#2]
**Summary**
Upon following the calibration step and repeating the gesture of "enter" the green bar won't fill, once in a while you'll get "gesture too different" which indicates that it's recognizing a gesture being made but the green bar doesn't fill to bring you on to the next step.

**Steps to reproduce**
1) Go to the Calibration application.
2) Scroll down to skip the initial welcome video.
3) Tap "START".
4) Follow the instructions as they are displayed on the screen as normal.

**Expected**
The green bar should fill upon the user making accurate gestures repetitively.

**Actual**
The green bar showing that an accurate gesture has been repeated enough times to register as a gesture isn't being filled and so doesn't move on to the next gesture. Sometimes a message appears showing "gesture too different" which means that it is actively listening for gestures.

**Possible Workaround**
If this issue occurs, swipe the screen to close the calibration application and start the calibration application again, this usually allows the user to continue as normal throughout the calibration.

**Screenshots**
![lhoward 18-jun-2016 - calibration bar doesn t fill upon accurate gesture repetition](https://cloud.githubusercontent.com/assets/13253346/16170312/bbc26068-3546-11e6-8d39-1cc8fcb50853.png)

**Comment by denni2007** [comment]
As for now, for the testing purposes, only gesture UP is filling the progress bar, DOWN in turn, means - "gesture too different!", HOME - "gesture too similar!"

**Comment by lhoward465** [comment]
@denni2007 I disagree, if that were the case then the message "gesture too similar" would appear every time the user was calibrating the "HOME" gesture, this is not the case currently. When the calibration application is functioning correct (as I perceive anyway) the bar fills upon each accurately repeated gesture. If I firstly make a gesture and then perform a different gesture the "Gesture too different" message then appears. (we may be using different versions)

**Comment by denni2007** [comment]
Sorry, maybe we are using different hardware, I will figure it out and reply you.
